docs/copied-from-beats/command-reference.asciidoc

```diff
@@ -82,31 +82,31 @@ endif::[]
 [options="header"]
 |=======================
 |Commands |
-ifeval::[("{beatname_lc}"=="functionbeat")]
+ifdef::functionbeat[]
```

Comment:
Single equality *should* work in asciidoctor.

Reply:
Good catch. They do work. I've removed them from this PR.